addons/web_tour/static/src/js/tour_step_utils.js

```diff
@@ -157,5 +157,43 @@ return Class.extend({
             },
         ].map(this.addDebugHelp(this._getHelpMessage('mobileKanbanSearchMany2X', modalTitle, valueSearched)));
     },
+    /**
+     * Utility steps to save a form and wait for the save to complete
+     *
+     * @param extra_trigger additional save-condition selector
+     */
+    saveForm(extra_trigger) {
+        return [{
+            content: "save form",
+            trigger: '.o_form_button_save:contains("Save")',
+            extra_trigger,
+            run: 'click',
+            auto: true,
+        }, {
+            content: "wait for save completion",
+            trigger: '.o_form_button_edit',
+            run() {},
+            auto: true,
+        }];
+    },
+    /**
+     * Utility steps to cancel a form creation or edition.
+     *
+     * Supports creation/edition from either a form or a list view (so checks
+     * for both states).
+     */
+    exitForm() {
+        return [{
+            content: "exit the form",
+            trigger: ".o_form_button_cancel",
+            run: 'click',
+            auto: true,
+        }, {
+            content: "wait for cancellation to complete",
+            trigger: "button.o_list_button_add, button.o_form_button_edit",
```

Comment:
I'd rather wait for ".o_form_readonly" (twice)

Reply:
@aab-odoo and `.o_list_renderer` for the list view rather than `o_list_button_add`?

Reply:
There's no such classname for the list, unfortunately

Reply:
Right but this helper is only to go from a genuine form view to a list view, it doesn't handle editable lists (I don't think the checker will trigger on those, which might eventually become an issue but...)